exporter/splunkhecexporter/logdata_to_splunk.go

```diff
@@ -140,6 +140,10 @@ func convertAttributeValue(value pdata.AttributeValue, logger *zap.Logger) inter
 }
 
 // nanoTimestampToEpochMilliseconds transforms nanoseconds into <sec>.<ms>. For example, 1433188255.500 indicates 1433188255 seconds and 500 milliseconds after epoch.
-func nanoTimestampToEpochMilliseconds(ts pdata.TimestampUnixNano) float64 {
-	return time.Duration(ts).Round(time.Millisecond).Seconds()
+func nanoTimestampToEpochMilliseconds(ts pdata.TimestampUnixNano) *float64 {
+	val := time.Duration(ts).Round(time.Millisecond).Seconds()
+	if val == 0 {
+		return nil
+	}
```

Comment:
Add a comment to explain why this is important.
Also, I think it is better to check for 0 before doing the conversion to seconds. Technically it may be 0.1 seconds which is not a missing timestamp but rounds to 0 seconds.

Reply:
Thanks, I've done this now